config/settings.yml

```diff
@@ -1081,6 +1081,10 @@
   :timeout: 120
   :use_vim_broker: true
   :authentication_timeout: 30.seconds
+  :remote_miq_api:
+    :user: admin
```

Comment:
I'm okay either way, I had a raise if the user wasn't set before, but removed it because we were setting a default.